<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TempPage</class>
 <widget class="QWidget" name="TempPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_midCenter2">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" rowstretch="4,4,2" columnstretch="1,1,1" rowminimumheight="0,0,0" columnminimumwidth="0,0,0">
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>30</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="card1" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="card4" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="card6" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="card5" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="QWidget" name="card3" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="card2" native="true"/>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
